### DISCUSSION TITLE
Scripting

### DISCUSSION THREAD

**Discussion by StoneyBricks** [rgriebl/brickstore#762] [General]
I might be a much older guy than most and never had much to do with coding in my life. But I see many saying that a function can be “scripted”. Can anyone of you tell me what type of unique functions can be scripted for the Brickstore and how they work/implicated?

**Comment by JulioC** [comment]
I'm also after this.

It looks like there's support for extensions, written in JavaScript or QML: https://www.brickstore.dev/extensions/index.html

The repository has a few examples: https://github.com/rgriebl/brickstore/tree/main/extensions
